fe/fe-core/src/main/java/com/starrocks/http/rest/TransactionLoadAction.java

```diff
@@ -158,6 +158,19 @@ public void executeTransaction(BaseRequest request, BaseResponse response) throw
                 resp.addResultEntry("Label", label);
                 sendResult(request, response, resp);
                 return;
+            } else if (txnStatus == TransactionStatus.COMMITTED || txnStatus == TransactionStatus.VISIBLE
+                      || txnStatus == TransactionStatus.ABORTED) {
+                String okMsg;
+                Long txnID = GlobalStateMgr.getCurrentGlobalTransactionMgr().getLabelTxnID(db.getId(), label);
+                if (txnID == -1) {
+                    okMsg = "label " + label + " txn already expired";
```

Comment:
when re-commit or re-rollback the transaction, and the transaction has already completed, `txnId NOT FOUND` is not error, just probably expried,   return  instead of throwing exception is acceptable